app/payment/migrations/0001_initial.py

```diff
@@ -11,39 +11,86 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
+        ("content", "0054_registration_payment_expiredate"),
```

Comment:
Ikke endre gamle migrasjonsfiler

Reply:
Her var det en initial migration, og så endret jeg noe på modellen. For å minimere antall migrasjonsfiler så slettet jeg begge migrasjonsfilene og kjørte makemigrations på nytt. Så jeg har ikke endret noe manuelt, er Django som har fikset.

Reply:
Dette kommer til å ødlegge databasen som martin sier, for denne komme ikke til å bli kjørt på migrate når den blir lagt til i prod. Heller kjør makemigration og la det bli flere

Reply:
Ja, merket det selv lokalt etter at jeg hadde kommentert, at det ble problemer. Jeg fikset det tilbake slik det var og bruke makemigration på nye endringer